core/capabilities/ccip/ccipsolana/gas_helpers.go

```diff
@@ -13,10 +13,10 @@ type EstimateProvider struct {
 
 // CalculateMerkleTreeGas is not implemented
 func (gp EstimateProvider) CalculateMerkleTreeGas(numRequests int) uint64 {
-	return 0
+	return 1
```

Comment:
I can't remember why this was necessary but the plugin didn't like it when the cost estimate ended up being zero

Reply:
Do we have the logs / errors for when this was set to 0?

It doesn't matter too much for Solana since we'll always execute 1msg/report; ideally that'd be clear when looking through this gashelpers code.

Reply:
I agree with @makramkd here. 1 is fine. Perhaps things could be improved in the future with a `NoOp` provider that can be shared with most/all of the non-EVMs.